ethereum/spec/src/main/java/tech/pegasys/teku/spec/config/SpecConfigEip4844.java

```diff
@@ -0,0 +1,43 @@
+/*
+ * Copyright ConsenSys Software Inc., 2022
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with
+ * the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on
+ * an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations under the License.
+ */
+
+package tech.pegasys.teku.spec.config;
+
+import java.util.Optional;
+import org.apache.tuweni.bytes.Bytes;
+
+public interface SpecConfigEip4844 extends SpecConfigCapella {
+  Bytes BLOB_TX_TYPE = Bytes.fromHexString("0x05");
```

Comment:
Even if this is not strictly a spec config, i see the benefits of having these constants being defined here instead of in `Constants` (which Adrian is tring to get the rid of it for a while :) )

Reply:
Looks like we resolved this